rewrite_external_links/middleware.py

```diff
@@ -48,7 +49,10 @@ def linkrepl(m):
                     link=urlencode(h.unescape(m.group('link')), safe=''),
                     after=m.group('after'),
                 )
-            response.content = self.extlinks.sub(linkrepl, response.content.decode())
+            response.content = self.extlinks.sub(
+                linkrepl,
+                response.content.decode('utf-8')
```

Comment:
I was wondering that.  Quite possibly.  I put some Greek letters in the URL in the test and they work, but I haven't investigated any further.